### DISCUSSION TITLE
Bravo Changes Won't show on my PBI Model

### DISCUSSION THREAD

**Discussion by egakagoc2xi** [sql-bi/Bravo#763] [Q&A]
Hi,

I realized this maybe a noob question, but whether I open the from externals tools or open the app first, whatever change I make trhough bravo, won't be reflected on the PBI Report.

Can someone please help me?

Thanks

**Comment by albertospelta** [comment]
The cause could be the one reported in https://github.com/TabularEditor/TabularEditor/issues/977 and https://github.com/sql-bi/Bravo/issues/179#issuecomment-1075284256.

We are aware of this issue where Power BI Desktop doesn't update when changes are made using an external tool. This is a problem that affects not only Bravo but also other external tools, including Tabular Editor.

As of today, the only workaround seems to be forcing Power BI Desktop to synchronize, for example, by creating a new measure. This is just a workaround, but it should work.
